src/snowflake/snowpark/dataframe.py

```diff
@@ -2204,6 +2210,7 @@ def pivot(
             snowflake.snowpark.relational_grouped_dataframe._PivotType(
                 pc[0], pivot_values, default_on_null
```

Comment:
nit: we might want to rename `ast_stmt` to `_ast_stmt` or so.

Reply:
Done.